service/controller/chart/v1/version_bundle.go

```diff
@@ -0,0 +1,20 @@
+package v1
+
+import (
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
+	return versionbundle.Bundle{
+		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "chart-operator",
+				Description: "Add your changes here.",
+				Kind:        versionbundle.KindAdded,
+			},
+		},
+		Components: []versionbundle.Component{},
+		Name:       "chart-operator",
+		Version:    "1.0.0",
```

Comment:
The operator needs to expose both chart and chartconfig version bundles. I don't want to add the filtering we have in cluster-operator because its complex and I don't think its needed.

`chart` version bundles will be `1.y.z` and `chartconfig` version bundles will be `0.y.z`.